### ISSUE TITLE
Alert when adding non-existing local file

### ISSUE THREAD

**Issue by M3ssman** [OCR-D/core#495]
Hello,
I'm wondering if it's really intended behavior that a non-existing local file is silently added to metsfile by `ocrd workspace add`:

```
ocrd workspace add -m "image/jpg" -G 'MAX' -i "IMG_MAX_1246734" "data/IMG_MAX_1246734.jpg"

=> mets.xml:

<mets:FLocat LOCTYPE="OTHER" OTHERLOCTYPE="FILE" xlink:href="data/IMG_MAX_1246734.jpg"/>
```
Is there a reason to do so? 
I'd expect an Exception in that case to halt ocr-ing, since any subsequent steps produce no valid output but do not complain either.

Version: OCRD V2.7.1
OS: Ubuntu 18.04 LTS

**Comment by kba** [comment]
Background: When we started `core`, the assumption was that all references to files should be URL, not necessarily pointing to local files. Since then we've pivoted towards using relative filenames everywhere. That is why `ocrd workspace add` does not assume the file argument to be a file.

However, I can understand how this can be confusing for developers. We can implement a file-exists check.

Should this be an opt-in (like a `--check-file-exists` flag) or an opt-out (e.g. `--no-check-file-exists)?

I'd prefer the former because backwards-compatibility.
